### ISSUE TITLE
Session 88e5eab8-83cb-4991-8281-5037d07a45e7. Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by yzarricueta** [Azure/App-Service-Migration-Assistant#24]
#### Investigative information

Please provide the following:

- UTC Timestamp: 15:30 PM 2020-01-06
- Migration Tool App version : Azure App Service Migration Tool V. 1.0.1
- OS Version: Windows Server 2008 R2 Standard

#### Repro steps

After I choose a site, the system cannot generate assessment report for web site. I attached the json report. Please tell me what is wrong?
[AnonAssessmentInfo.json.txt](https://github.com/Azure/App-Service-Migration-Assistant/files/4026778/AnonAssessmentInfo.json.txt)

Provide the steps required to reproduce the problem:

#### Expected behavior

Provide a description of the expected behavior.

#### Actual behavior

Provide a description of the actual behavior observed.

#### Known workarounds

Provide a description of any known workarounds.

**Comment by wadeh-ms** [comment]
Hi!

Thanks for much for sending this file.  I was able to use it to reproduce the problem here and find the bug.

The issue is that some configuration objects in the JSON are not deserializing properly.  This result in invalid objects in the analysis code that we do here, which leads to the exception.

This is something that we can fix on our side, without anyone needing to re-download the Migration Assistant.

Once we have made the fix here, I will let you know, so you can try it again.

I do want to set your expectations, though, that once this is fixed, your site will have some issues which will need to be addressed before it could be automatically migrated.  Once the fix is in place, the behavior change that you should see, is that the Migration Assistant will list them for you, hopefully with enough information to understand how to proceed.

We would be interested in your feedback about that, once the fix for the current issue is available.

Thanks again,
-Wade

**Comment by yzarricueta** [comment]
Perfect, thank you for your fast answer. 
I will be waiting the fix, as soon as possible.

THXS.

**Comment by krolson** [comment]
We have updated the assessment site and it should now be able to return the full assessment results to you. Please give it another try and let us know if you are still seeing any issues or have questions/comments about the assessment results.

And thank you, again, for providing the debugging details!

-Kristina

**Comment by krolson** [comment]
We believe this issue should be fixed as of last messages - please open a new issue if you see this behavior still occurring.
